test/intern.js

```diff
@@ -12,6 +12,16 @@ define({
     host: 'localhost',
     port: 4444
   },
+  kibana: {
+    protocol: 'http',
+    hostname: 'localhost',
+    port: 5620
+  },
+  elasticsearch: {
+    protocol: 'http',
+    hostname: 'localhost',
+    port: 9220
```

Comment:
Hmmm, I changed this config value, and when I ran the server, it was still on port 9220. Am I doing something wrong?